client/components/SongRow/index.styl

```diff
@@ -1,105 +1,85 @@
-@import "~/stylus/color.styl"
-@import "~/stylus/sizes.styl"
+@import "~/stylus/settings/color.styl"
+@import "~/stylus/settings/sizes.styl"
 
 .songRow
-  border-radius: 3px
-  display grid
-  cursor pointer 
-  grid-template-columns: 42px 1fr auto auto 42px;
-  grid-template-rows: auto auto
-  grid-auto-flow: column;
-  grid-column-gap: 10px
-  align-items center
-  padding 10px 0
-
-  @media all and (min-width: size-medium)
-    grid-template-rows: 1fr
-    padding 0px 0
-
-  .heartIconWrapper
-    width 100%
+    border-radius: 3px
+    cursor pointer
+    display flex
+    flex-direction column
+    justify-content space-between
+
+    @media all and (min-width: size-medium)
+        flex-direction row
+
+.songActions
     display flex
-    display: none // GRID TEMP
     align-items center
-    &.fav
-      filter: invert(66%) sepia(95%) saturate(2455%) hue-rotate(359deg) brightness(101%) contrast(107%);
-      &:hover
-        filter none
-    &:not(.fav):hover
-      filter: invert(66%) sepia(95%) saturate(2455%) hue-rotate(359deg) brightness(101%) contrast(107%);
-    
-    img
-      width 100%
-
-  &:nth-of-type(odd)
-    background-color rgba(colorGray800, 0.3)
-
-  @media (any-hover: hover)
-    &:hover 
-      box-shadow 0 0 0 2px colorAmber800
-      position: relative
-
-  &.playing
-    color colorAmber500
-
-a
-  @media (any-hover: hover)
-    &:hover 
-      color colorAmber500
+    margin-left auto
+
+    @media all and (min-width: size-medium)
+        & > *
```

Comment:
This is hard to understand at face value, what is the purpose of this type of selector.